website/docs/prysm-usage/checkpoint-sync.md

```diff
@@ -46,18 +81,20 @@ INFO[0001] saved ssz-encoded block to to block_prater_altair_3041920-0x766bdce4c
 INFO[0001] saved ssz-encoded state to to state_prater_altair_3041920-0x34ebc10f191706afbbccb0c3c39679632feef0453fe842bda264e432e9e31011.ssz
 ```
 
-For human-friendliness, the file name includes the ssz type (`state`, `block`), the network (`prater`), the fork name (`altair`), the slot (`2397120`) and the state or block root in hex encoding. The checkpoint save command does not download the required genesis state at this time, but this can be easily downloaded via `curl` or `wget` using the `genesis` named state identifier:
+<!--meta: I imagine most readers don't know what ssz means, and won't care. We can elaborate on the technical details in dev wiki > dev concepts if/when needed.-->
+<!--meta: contractions can make guidance a bit friendlier/natural, which can make content more readable, which can reduce the cognitive cost of learning/doing -->
+<!--meta: in general we want to avoid claiming that a task is "easy" - some readers may not find it easy at all. -->
+
+The exported file name includes the file type (`state`, `block`), the network (`prater`), the fork name (`altair`), the slot (`2397120`) and the state or block root in hex encoding. The checkpoint save command doesn't export the required genesis state, but this can be downloaded via `curl` or `wget` using the following command syntax:
```

Comment:
should we revisit this? prater is now goerli and fork will be bellatrix 🤔

Reply:
oooh good catch, mind dropping your suggestion via a `suggested commit` comment while I tidy up / fix build? That will let me click a button to apply your suggestion 🙏